blueprints/common/partials/imagebuilder_infrastructureconfiguration.hbs

```diff
@@ -0,0 +1,62 @@
+{{!--
+  Creates an Image Builder Infrastructure Configuration
+https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-imagebuilder-infrastructureconfiguration.html
+--}}
+
+"imagebuilder{{sanitize this.Name ''}}":{
+  "Type" : "AWS::ImageBuilder::InfrastructureConfiguration",
+  "Properties" : {
+
+      {{#if this.Description}}
+        "Description" : "{{this.Description}}",
+      {{/if}}
+
+        "Name" : "{{this.Name}}",
+
+        "InstanceProfileName" : {{#ifObject this.InstanceProfileName}}{ {{{inject this.InstanceProfileName}}} }{{else}}"{{this.InstanceProfileName}}"{{/ifObject}},
+
+      {{#if this.KeyPair}}
+        "KeyPair" : "{{this.KeyPair}}",
+      {{/if}}
+
+      {{#if this.SnsTopicArn}}
+        "SnsTopicArn" : "{{this.SnsTopicArn}}",
+      {{/if}}
+
+      {{#if this.SubnetId}}
+        "SubnetId" : "{{this.SubnetId}}",
```

Comment:
I generally try to help a bit more with this type of argument.  I don't know exactly how this component works but having users "hardcode" an Id is generally a bad thing.  In this case the `SubnetId` is different between environments so the blueprints won't be portable.  The good news is that the subnet information is already in the environment global file.

Take a look at the ELBv2 LoadBalancer (blueprints/common/partials/elasticloadbalancingv2_loadbalancer.hbs) partial for an extreme example of bending over backwards to help a user.